components/searchable_user_list/searchable_user_list.jsx

```diff
@@ -127,7 +127,8 @@ export default class SearchableUserList extends React.Component {
             endCount = -1;
         } else {
             startCount = this.props.page * this.props.usersPerPage;
-            endCount = startCount + count;
+            endCount = startCount + this.props.usersPerPage;
```

Comment:
Could you maybe simplify this into `Math.minimum(startCount + this.props.usersPerPage, total)` to make it a bit clearer what's happening?

Reply:
@hmhealey Good call, fixed!